doc/about.rst

```diff
@@ -86,7 +86,7 @@ If you want to cite scikit-learn for its API or design, you may also want to con
 following paper:
 
   `API design for machine learning software: experiences from the scikit-learn
-  project <https://arxiv.org/abs/1309.0238>`_, Buitinck *et al.*, 2013.
+  project <:arxiv:`1309.0238`>`_, Buitinck *et al.*, 2013.
```

Comment:
```suggestion
  project :arxiv:`1309.0238`, Buitinck *et al.*, 2013.
```
If you check the [rendering](https://154018-843222-gh.circle-artifacts.com/0/doc/about.html) of this pull request you see that the references are not displayed correctly. You might want to check [scipy/#12858](https://github.com/scipy/scipy/pull/12858) to see how `doi` and `arxiv` directives should be used.  My suggestion here is an example.

Reply:
Thanks, let me check :)